resources/views/index.blade.php

```diff
@@ -0,0 +1,44 @@
+@extends('layout')
+@section('content')
+    @if ($message = Session::get('success'))
```

Comment:
```suggestion
    @if ($message == Session::get('success'))
```
message này là gì e em?

Reply:
em cho 1 cái thông báo để khi thêm,sửa hay xóa thì nó hiện ra ở trên đầu ạ